docs/consensus/proof_of_space.md

```diff
@@ -72,20 +73,18 @@ Computes the 7 PoS tables.
     6. Map the match pair entries `(left_entry, right_entry)` positions back from the bucket to their respective positions in the previous table, `left_position` and `right_position`.
     7. Push to the current table an entry `(ys, metadata, positions) = (y, c, {left_position, right_position})`
     8. Sort the current table by `y` value.
-        <!-- 
-        TODO: toggle
-        Note
-            
-            chiapos and spaceframe use k-way merge to deal with the fact that table might be too large to store in memory. Since we don’t have this problem we could use a faster algorithm. -->
+
+        <Collapsible title="Note">
+        chiapos and spaceframe use k-way merge to deal with the fact that table might be too large to store in memory. Since we don’t have this problem we could use a faster algorithm.
+        </Collapsible>
             
 
 The resulting `pos_tables` is a vector containing the 7 tables in order.
 
-<!-- 
-TODO: replace diagram
-![Steps 3.b and 3.c in Generate. Numbers and matching function $M$ are solely illustrative. Since the `entry.y` is sorted, Chia does this sequentially on portions of the table as their tables don’t fit in memory.  Notice that 235 and 380 appear in multiple matches, while some numbers don’t have a match.](Dilithium%20PoS%20Specification%2008339021db504ce9aa0be6cc2f52a949/Subspace_v2_Master_-_Plotting_(9).png)
+![Steps 3.b and 3.c in Generate. Numbers and matching function $M$ are solely illustrative. Since the `entry.y` is sorted, Chia does this sequentially on portions of the table as their tables don’t fit in memory.  Notice that 235 and 380 appear in multiple matches, while some numbers don’t have a match.](/img/Proof_of_space_plotting.png)
```

Comment:
```suggestion
![Steps 4.ii-iv in generate function definition. Numbers and matching function $M$ are solely illustrative. Since the `entry.y` is sorted, Chia does this sequentially on portions of the table as their tables don’t fit in memory.  Notice that 235 and 380 appear in multiple matches, while some numbers don’t have a match.](/img/Proof_of_space_plotting.png)
```